go.mod

```diff
@@ -23,6 +23,7 @@ require (
 	github.com/youmark/pkcs8 v0.0.0-20181117223130-1be2e3e5546d
 	golang.org/x/crypto v0.0.0-20201216223049-8b5274cf687f
 	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
+	golang.org/x/text v0.3.7 // indirect
```

Comment:
Shall we also comment on the dependency like [#999](https://github.com/mongodb/mongo-go-driver/pull/999)?